Use new terms enum API for autocomplete value suggestions

lukasolson opened the pull request:
## Summary

Requires https://github.com/elastic/kibana/pull/100586.
Resolves https://github.com/elastic/kibana/issues/92783.
Resolves https://github.com/elastic/kibana/issues/97512.

Uses the new [terms enum](https://github.com/elastic/elasticsearch/pull/66452) API for a faster autocomplete experience.

Note that currently, the autocomplete only targets warm and hot data tiers. By default, indices are given `data_content` tier, so to update this, you need to do something like this:

```
PUT /my-index-*/_settings
{
  "index.routing.allocation.include._tier_preference": "data_hot"
}
```

### To do

- [x] Config for timeout, data tiers, A/B testing?
- [x] Update unit/functional tests

### Checklist

- [ ] [Documentation](https://www.elastic.co/guide/en/kibana/master/development-documentation.html) was added for features that require explanation or tutorials
- [ ] [Unit or functional tests](https://www.elastic.co/guide/en/kibana/master/development-tests.html) were updated or added to match the most common scenarios
- [ ] If a plugin configuration key changed, check if it needs to be allowlisted in the [cloud](https://github.com/elastic/cloud) and added to the [docker list](https://github.com/elastic/kibana/blob/c29adfef29e921cc447d2a5ed06ac2047ceab552/src/dev/build/tasks/os_packages/docker_generator/resources/bin/kibana-docker)

### Release note

Value suggestions for KQL autocomplete and creating filters now use the Elasticsearch _terms_enum API for improved performance.

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/126123/)
* Commit: 36bad6e3f4998f3c69f8ce4811968ef919c308e8
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [API Integration Tests](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/126123/execution/node/386/log)

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/126123/","number":"126123","commit":"36bad6e3f4998f3c69f8ce4811968ef919c308e8"}]}
PIPELINE-->

markharwood commented:
>Note that currently, the autocomplete only targets warm and hot data tiers. 

A heads up on how the API evolved - rather than hard-coding that tier filtering logic into the terms_enum API we changed that and made filtering a responsibility of the calling client. The client should filter indices by using a query on the new `_tier` metadata field so a typical Kibana query would look like: 
```
localhost:9200/myindex/_terms_enum
{
    "field" : "myfield",
    "string" : "Microsof",
    "index_filter": {
        "bool":{
            "must":[
               { "range": "my Kibana time-picker range"},
               { "terms": {"_tier": ["data_warm", "data_hot"]}
             ]
        }
    }
}
```
Note that the queries used above are a rough filter on whole indices and not individual documents when it comes to where we look for term suggestions (i.e. we might suggest terms originating from docs outside of the provided date range if the docs sit in an index whose time span partially matches the index filter).

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/126930/)
* Commit: 16a451d0daa3ca6905af881d29804adbaa3b77c9
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [API Integration Tests](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/126930/execution/node/390/log)

### History
* :broken_heart: [Build #126123](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/126123/) failed 36bad6e3f4998f3c69f8ce4811968ef919c308e8

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/126123/","number":"126123","commit":"36bad6e3f4998f3c69f8ce4811968ef919c308e8"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/126930/","number":"126930","commit":"16a451d0daa3ca6905af881d29804adbaa3b77c9"}]}
PIPELINE-->

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/127286/)
* Commit: c6307fb3ab8f6df74884e7b7fb1bb3197549bddf
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [API Integration Tests](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/127286/execution/node/386/log)

### History
* :broken_heart: [Build #126930](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/126930/) failed 16a451d0daa3ca6905af881d29804adbaa3b77c9
* :broken_heart: [Build #126123](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/126123/) failed 36bad6e3f4998f3c69f8ce4811968ef919c308e8

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/126123/","number":"126123","commit":"36bad6e3f4998f3c69f8ce4811968ef919c308e8"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/126930/","number":"126930","commit":"16a451d0daa3ca6905af881d29804adbaa3b77c9"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/127286/","number":"127286","commit":"c6307fb3ab8f6df74884e7b7fb1bb3197549bddf"}]}
PIPELINE-->

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/127802/)
* Commit: fd57f552107db4fbbf4d9455fd11a272461a6f51
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [API Integration Tests](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/127802/execution/node/380/log)

### History
* :broken_heart: [Build #127286](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/127286/) failed c6307fb3ab8f6df74884e7b7fb1bb3197549bddf
* :broken_heart: [Build #126930](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/126930/) failed 16a451d0daa3ca6905af881d29804adbaa3b77c9
* :broken_heart: [Build #126123](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/126123/) failed 36bad6e3f4998f3c69f8ce4811968ef919c308e8

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/126123/","number":"126123","commit":"36bad6e3f4998f3c69f8ce4811968ef919c308e8"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/126930/","number":"126930","commit":"16a451d0daa3ca6905af881d29804adbaa3b77c9"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/127286/","number":"127286","commit":"c6307fb3ab8f6df74884e7b7fb1bb3197549bddf"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/127802/","number":"127802","commit":"fd57f552107db4fbbf4d9455fd11a272461a6f51"}]}
PIPELINE-->

lukasolson commented:
> A heads up on how the API evolved - rather than hard-coding that tier filtering logic into the terms_enum API we changed that and made filtering a responsibility of the calling client. The client should filter indices by using a query on the new `_tier` metadata field so a typical Kibana query would look like:

@markharwood I've added it as a kibana.yml setting and defaulted it to `["data_hot", "data_warm"]`. Does this seem like a reasonable default, or should we include `data_content` (since nodes default to that tier)?

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128440/)
* Commit: cb0ef5eecf4854a4111c7651830f7ba257bd0f7d
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [API Integration Tests](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128440/execution/node/410/log)

### History
* :broken_heart: [Build #127802](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/127802/) failed fd57f552107db4fbbf4d9455fd11a272461a6f51
* :broken_heart: [Build #127286](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/127286/) failed c6307fb3ab8f6df74884e7b7fb1bb3197549bddf
* :broken_heart: [Build #126930](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/126930/) failed 16a451d0daa3ca6905af881d29804adbaa3b77c9
* :broken_heart: [Build #126123](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/126123/) failed 36bad6e3f4998f3c69f8ce4811968ef919c308e8

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/126123/","number":"126123","commit":"36bad6e3f4998f3c69f8ce4811968ef919c308e8"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/126930/","number":"126930","commit":"16a451d0daa3ca6905af881d29804adbaa3b77c9"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/127286/","number":"127286","commit":"c6307fb3ab8f6df74884e7b7fb1bb3197549bddf"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/127802/","number":"127802","commit":"fd57f552107db4fbbf4d9455fd11a272461a6f51"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128440/","number":"128440","commit":"cb0ef5eecf4854a4111c7651830f7ba257bd0f7d"}]}
PIPELINE-->

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128536/)
* Commit: 8e81a0d0e257b95ac174f01710cb54b4ac6a6bed
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [API Integration Tests](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128536/execution/node/419/log)

### History
* :broken_heart: [Build #128440](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128440/) failed cb0ef5eecf4854a4111c7651830f7ba257bd0f7d
* :broken_heart: [Build #127802](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/127802/) failed fd57f552107db4fbbf4d9455fd11a272461a6f51
* :broken_heart: [Build #127286](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/127286/) failed c6307fb3ab8f6df74884e7b7fb1bb3197549bddf
* :broken_heart: [Build #126930](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/126930/) failed 16a451d0daa3ca6905af881d29804adbaa3b77c9
* :broken_heart: [Build #126123](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/126123/) failed 36bad6e3f4998f3c69f8ce4811968ef919c308e8

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/126123/","number":"126123","commit":"36bad6e3f4998f3c69f8ce4811968ef919c308e8"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/126930/","number":"126930","commit":"16a451d0daa3ca6905af881d29804adbaa3b77c9"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/127286/","number":"127286","commit":"c6307fb3ab8f6df74884e7b7fb1bb3197549bddf"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/127802/","number":"127802","commit":"fd57f552107db4fbbf4d9455fd11a272461a6f51"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128440/","number":"128440","commit":"cb0ef5eecf4854a4111c7651830f7ba257bd0f7d"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128536/","number":"128536","commit":"8e81a0d0e257b95ac174f01710cb54b4ac6a6bed"}]}
PIPELINE-->

markharwood commented:
>defaulted it to ["data_hot", "data_warm"]. Does this seem like a reasonable default, or should we include data_content (since nodes default to that tier)?

adding `data_content` sounds like it would make sense, covering those indices that don't "age out". Can you confirm @jimczi ?

jimczi commented:
+1 to add `data_content` by default. I also wonder if `data_cold` should be considered. Are we applying the active date range filter on every request ? If so, then it would be nice to consider `data_cold` as default too since most of the requests to that tier would be filtered by the range. We have the data locally in that tier so I think it makes sense to provide suggestions as long as we can filtered the amount of queries that we need to handle (with the active range filter).

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128722/)
* Commit: 6634748ad835946b4dff864ef43006653ff8c97f
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-100174/6634748ad835946b4dff864ef43006653ff8c97f)
* This comment will update when the build is complete

### Failed CI Steps
* [Execute xpack-kibana-ciGroup9](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128722/execution/node/738/log)
* [Execute xpack-kibana-ciGroup9](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128722/execution/node/831/log)

### History
* :broken_heart: [Build #128536](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128536/) failed 8e81a0d0e257b95ac174f01710cb54b4ac6a6bed
* :broken_heart: [Build #128440](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128440/) failed cb0ef5eecf4854a4111c7651830f7ba257bd0f7d
* :broken_heart: [Build #127802](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/127802/) failed fd57f552107db4fbbf4d9455fd11a272461a6f51
* :broken_heart: [Build #127286](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/127286/) failed c6307fb3ab8f6df74884e7b7fb1bb3197549bddf
* :broken_heart: [Build #126930](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/126930/) failed 16a451d0daa3ca6905af881d29804adbaa3b77c9

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/126930/","number":"126930","commit":"16a451d0daa3ca6905af881d29804adbaa3b77c9"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/127286/","number":"127286","commit":"c6307fb3ab8f6df74884e7b7fb1bb3197549bddf"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/127802/","number":"127802","commit":"fd57f552107db4fbbf4d9455fd11a272461a6f51"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128440/","number":"128440","commit":"cb0ef5eecf4854a4111c7651830f7ba257bd0f7d"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128536/","number":"128536","commit":"8e81a0d0e257b95ac174f01710cb54b4ac6a6bed"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128722/","number":"128722","commit":"6634748ad835946b4dff864ef43006653ff8c97f"}]}
PIPELINE-->

elasticmachine commented:
Pinging @elastic/kibana-app-services (Team:AppServices)

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/130787/)
* Commit: 9896f5980125027282a1a094fa179f7c97e5e8c9
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [Check Doc API Changes](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/130787/execution/node/425/log)

### History
* :broken_heart: [Build #128722](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128722/) failed 6634748ad835946b4dff864ef43006653ff8c97f
* :broken_heart: [Build #128536](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128536/) failed 8e81a0d0e257b95ac174f01710cb54b4ac6a6bed
* :broken_heart: [Build #128440](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128440/) failed cb0ef5eecf4854a4111c7651830f7ba257bd0f7d
* :broken_heart: [Build #127802](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/127802/) failed fd57f552107db4fbbf4d9455fd11a272461a6f51
* :broken_heart: [Build #127286](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/127286/) failed c6307fb3ab8f6df74884e7b7fb1bb3197549bddf

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/127286/","number":"127286","commit":"c6307fb3ab8f6df74884e7b7fb1bb3197549bddf"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/127802/","number":"127802","commit":"fd57f552107db4fbbf4d9455fd11a272461a6f51"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128440/","number":"128440","commit":"cb0ef5eecf4854a4111c7651830f7ba257bd0f7d"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128536/","number":"128536","commit":"8e81a0d0e257b95ac174f01710cb54b4ac6a6bed"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128722/","number":"128722","commit":"6634748ad835946b4dff864ef43006653ff8c97f"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/130787/","number":"130787","commit":"9896f5980125027282a1a094fa179f7c97e5e8c9"}]}
PIPELINE-->

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/130788/)
* Commit: 6812011de0316f5c7b6202033a57faba794f0ad5
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-100174/6812011de0316f5c7b6202033a57faba794f0ad5)
* This comment will update when the build is complete

### Failed CI Steps
* [Execute xpack-kibana-ciGroup9](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/130788/execution/node/764/log)
* [Execute xpack-kibana-ciGroup9](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/130788/execution/node/820/log)

### History
* :broken_heart: [Build #130787](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/130787/) failed 9896f5980125027282a1a094fa179f7c97e5e8c9
* :broken_heart: [Build #128722](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128722/) failed 6634748ad835946b4dff864ef43006653ff8c97f
* :broken_heart: [Build #128536](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128536/) failed 8e81a0d0e257b95ac174f01710cb54b4ac6a6bed
* :broken_heart: [Build #128440](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128440/) failed cb0ef5eecf4854a4111c7651830f7ba257bd0f7d
* :broken_heart: [Build #127802](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/127802/) failed fd57f552107db4fbbf4d9455fd11a272461a6f51

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/127802/","number":"127802","commit":"fd57f552107db4fbbf4d9455fd11a272461a6f51"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128440/","number":"128440","commit":"cb0ef5eecf4854a4111c7651830f7ba257bd0f7d"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128536/","number":"128536","commit":"8e81a0d0e257b95ac174f01710cb54b4ac6a6bed"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128722/","number":"128722","commit":"6634748ad835946b4dff864ef43006653ff8c97f"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/130787/","number":"130787","commit":"9896f5980125027282a1a094fa179f7c97e5e8c9"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/130788/","number":"130788","commit":"6812011de0316f5c7b6202033a57faba794f0ad5"}]}
PIPELINE-->

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133398/)
* Commit: 741add881de767511db2400082851cabfabff2df
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-100174/741add881de767511db2400082851cabfabff2df)
* This comment will update when the build is complete

### Failed CI Steps
* [Execute xpack-kibana-ciGroup9](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133398/execution/node/730/log)
* [Execute oss-pluginFunctional](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133398/execution/node/592/log)

### History
* :broken_heart: [Build #130788](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/130788/) failed 6812011de0316f5c7b6202033a57faba794f0ad5
* :broken_heart: [Build #130787](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/130787/) failed 9896f5980125027282a1a094fa179f7c97e5e8c9
* :broken_heart: [Build #128722](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128722/) failed 6634748ad835946b4dff864ef43006653ff8c97f
* :broken_heart: [Build #128536](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128536/) failed 8e81a0d0e257b95ac174f01710cb54b4ac6a6bed
* :broken_heart: [Build #128440](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128440/) failed cb0ef5eecf4854a4111c7651830f7ba257bd0f7d

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128440/","number":"128440","commit":"cb0ef5eecf4854a4111c7651830f7ba257bd0f7d"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128536/","number":"128536","commit":"8e81a0d0e257b95ac174f01710cb54b4ac6a6bed"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128722/","number":"128722","commit":"6634748ad835946b4dff864ef43006653ff8c97f"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/130787/","number":"130787","commit":"9896f5980125027282a1a094fa179f7c97e5e8c9"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/130788/","number":"130788","commit":"6812011de0316f5c7b6202033a57faba794f0ad5"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133398/","number":"133398","commit":"741add881de767511db2400082851cabfabff2df"}]}
PIPELINE-->

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133426/)
* Commit: ef001ef0e0b3b0c40070213d92a062de52420b64
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-100174/ef001ef0e0b3b0c40070213d92a062de52420b64)
* This comment will update when the build is complete

### Failed CI Steps
* [Execute xpack-kibana-ciGroup9](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133426/execution/node/726/log)
* [Execute xpack-kibana-ciGroup9](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133426/execution/node/821/log)

### History
* :broken_heart: [Build #133398](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133398/) failed 741add881de767511db2400082851cabfabff2df
* :broken_heart: [Build #130788](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/130788/) failed 6812011de0316f5c7b6202033a57faba794f0ad5
* :broken_heart: [Build #130787](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/130787/) failed 9896f5980125027282a1a094fa179f7c97e5e8c9
* :broken_heart: [Build #128722](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128722/) failed 6634748ad835946b4dff864ef43006653ff8c97f
* :broken_heart: [Build #128536](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128536/) failed 8e81a0d0e257b95ac174f01710cb54b4ac6a6bed

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128536/","number":"128536","commit":"8e81a0d0e257b95ac174f01710cb54b4ac6a6bed"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128722/","number":"128722","commit":"6634748ad835946b4dff864ef43006653ff8c97f"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/130787/","number":"130787","commit":"9896f5980125027282a1a094fa179f7c97e5e8c9"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/130788/","number":"130788","commit":"6812011de0316f5c7b6202033a57faba794f0ad5"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133398/","number":"133398","commit":"741add881de767511db2400082851cabfabff2df"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133426/","number":"133426","commit":"ef001ef0e0b3b0c40070213d92a062de52420b64"}]}
PIPELINE-->

Dosant commented:
@lukasolson, afaik it is not necessary to update `api_docs` as part of the pr.

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133666/)
* Commit: 0ef8b9440356621e493dfa36b5e43d2a620d750c
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-100174/0ef8b9440356621e493dfa36b5e43d2a620d750c)
* This comment will update when the build is complete

### Failed CI Steps
* [Execute xpack-kibana-ciGroup6](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133666/execution/node/716/log)
* [Execute xpack-kibana-ciGroup6](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133666/execution/node/953/log)

### History
* :broken_heart: [Build #133426](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133426/) failed ef001ef0e0b3b0c40070213d92a062de52420b64
* :broken_heart: [Build #133398](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133398/) failed 741add881de767511db2400082851cabfabff2df
* :broken_heart: [Build #130788](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/130788/) failed 6812011de0316f5c7b6202033a57faba794f0ad5
* :broken_heart: [Build #130787](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/130787/) failed 9896f5980125027282a1a094fa179f7c97e5e8c9
* :broken_heart: [Build #128722](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128722/) failed 6634748ad835946b4dff864ef43006653ff8c97f

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128722/","number":"128722","commit":"6634748ad835946b4dff864ef43006653ff8c97f"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/130787/","number":"130787","commit":"9896f5980125027282a1a094fa179f7c97e5e8c9"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/130788/","number":"130788","commit":"6812011de0316f5c7b6202033a57faba794f0ad5"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133398/","number":"133398","commit":"741add881de767511db2400082851cabfabff2df"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133426/","number":"133426","commit":"ef001ef0e0b3b0c40070213d92a062de52420b64"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133666/","number":"133666","commit":"0ef8b9440356621e493dfa36b5e43d2a620d750c"}]}
PIPELINE-->

nreese reviewed (approved):
maps changes LGTM
code review

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133879/)
* Commit: 056c7d097c258edea6db7b91acae684d1190701b
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-100174/056c7d097c258edea6db7b91acae684d1190701b)
* [Documentation Changes](https://kibana_100174.docs-preview.app.elstc.co/diff)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
> ✅ unchanged

### History
* :broken_heart: [Build #133666](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133666/) failed 0ef8b9440356621e493dfa36b5e43d2a620d750c
* :broken_heart: [Build #133426](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133426/) failed ef001ef0e0b3b0c40070213d92a062de52420b64
* :broken_heart: [Build #133398](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133398/) failed 741add881de767511db2400082851cabfabff2df
* :broken_heart: [Build #130788](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/130788/) failed 6812011de0316f5c7b6202033a57faba794f0ad5
* :broken_heart: [Build #130787](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/130787/) failed 9896f5980125027282a1a094fa179f7c97e5e8c9

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/130787/","number":"130787","commit":"9896f5980125027282a1a094fa179f7c97e5e8c9"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/130788/","number":"130788","commit":"6812011de0316f5c7b6202033a57faba794f0ad5"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133398/","number":"133398","commit":"741add881de767511db2400082851cabfabff2df"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133426/","number":"133426","commit":"ef001ef0e0b3b0c40070213d92a062de52420b64"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133666/","number":"133666","commit":"0ef8b9440356621e493dfa36b5e43d2a620d750c"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133879/","number":"133879","commit":"056c7d097c258edea6db7b91acae684d1190701b"}]}
PIPELINE-->

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/135233/)
* Commit: b45b4649efce09d58aeac4087dcb0410f21a5c7d
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-100174/b45b4649efce09d58aeac4087dcb0410f21a5c7d)
* [Documentation Changes](https://kibana_100174.docs-preview.app.elstc.co/diff)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
> ✅ unchanged

### History
* :green_heart: [Build #133879](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133879/) succeeded 056c7d097c258edea6db7b91acae684d1190701b
* :broken_heart: [Build #133666](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133666/) failed 0ef8b9440356621e493dfa36b5e43d2a620d750c
* :broken_heart: [Build #133426](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133426/) failed ef001ef0e0b3b0c40070213d92a062de52420b64
* :broken_heart: [Build #133398](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133398/) failed 741add881de767511db2400082851cabfabff2df
* :broken_heart: [Build #130788](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/130788/) failed 6812011de0316f5c7b6202033a57faba794f0ad5

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/130788/","number":"130788","commit":"6812011de0316f5c7b6202033a57faba794f0ad5"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133398/","number":"133398","commit":"741add881de767511db2400082851cabfabff2df"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133426/","number":"133426","commit":"ef001ef0e0b3b0c40070213d92a062de52420b64"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133666/","number":"133666","commit":"0ef8b9440356621e493dfa36b5e43d2a620d750c"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133879/","number":"133879","commit":"056c7d097c258edea6db7b91acae684d1190701b"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/135233/","number":"135233","commit":"b45b4649efce09d58aeac4087dcb0410f21a5c7d"}]}
PIPELINE-->

mattkime reviewed (approved):
code change look good, work well!

kibanamachine commented:
## 💔 Backport failed

| Status | Branch | Result |
|:------:|:------:|:------:|
| ❌ |  7.x  | Commit could not be cherrypicked due to conflicts |

To backport manually run:
`node scripts/backport --pr 100174`
